am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +126,41 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+
+    /**
+     * Returns a map of JsonPointers and set of permissions when record with user role and type exists in database.
+     *
+     * @param serviceName  (service_name)
+     * @param resourceType (resource_type)
+     * @param resourceName (resource_name)
+     * @param roleNames    A list of (role_names)
+     * @return permissionsByTypeAndRole or null
+     */
+    public Map<JsonPointer, Set<Permission>> getRolePermissions(
+        String serviceName, String resourceType, String resourceName, @NonNull List<String> roleNames) {
+        if (roleNames.size() > ROLENAMESIZE) {
```

Comment:
What the `ROLENAMESIZE` variable is for? It is absolutely ambitious to me. Please correct name.

Reply:
👍